README.md

```diff
@@ -32,12 +32,13 @@ Daily Builds
 
 |   Platform   |   Master<br>[![][build-badge-master]][build-master]   |   Release/1.1.X<br>[![][build-badge-1.X.X]][build-1.X.X]   |   Release/1.0.X<br>[![][build-badge-1.0.X]][build-1.0.X]   |
 |---------|:----------:|:----------:|:----------:|
-|   **Windows (x64)**    |   [![][win-x64-badge-master]][win-x64-version-master]<br>[Installer][win-x64-installer-master]<br>[zip][win-x64-zip-master]<br>[Symbols (zip)][win-x64-symbols-zip-master]   |   [![][win-x64-badge-1.1.X]][win-x64-version-1.1.X]<br>[Installer][win-x64-installer-1.1.X]<br>[zip][win-x64-zip-1.1.X]   |   [![][win-x64-badge-preview]][win-x64-version-preview]<br>[Installer][win-x64-installer-preview]<br>[zip][win-x64-zip-preview]   |
-|   **Windows (x86)**    |   [![][win-x86-badge-master]][win-x86-version-master]<br>[Installer][win-x86-installer-master]<br>[zip][win-x86-zip-master]<br>[Symbols (zip)][win-x86-symbols-zip-master]   |   [![][win-x86-badge-1.1.X]][win-x86-version-1.1.X]<br>[Installer][win-x86-installer-1.1.X]<br>[zip][win-x86-zip-1.1.X]   |   [![][win-x86-badge-preview]][win-x86-version-preview]<br>[Installer][win-x86-installer-preview]<br>[zip][win-x86-zip-preview]   |
-|   **Windows (arm32)**  |   [![][win-arm-badge-master]][win-arm-version-master]<br>[zip][win-arm-zip-master]<br>[Symbols (zip)][win-arm-symbols-zip-master]   |   N/A   |   N/A   |
-|   **Windows (arm64)**  |   [![][win-arm64-badge-master]][win-arm64-version-master]<br>[zip][win-arm64-zip-master]<br>[Symbols (zip)][win-arm64-symbols-zip-master]   |   N/A   |   N/A   |
-|   **Mac OS X (x64)**       |   [![][osx-badge-master]][osx-version-master]<br>[Installer][osx-installer-master]<br>[tar.gz][osx-targz-master]<br>[Symbols (tar.gz)][osx-symbols-targz-master]   |   [![][osx-badge-1.1.X]][osx-version-1.1.X]<br>[Installer][osx-installer-1.1.X]<br>[tar.gz][osx-targz-1.1.X]   |   [![][osx-badge-preview]][osx-version-preview]<br>[Installer][osx-installer-preview]<br>[tar.gz][osx-targz-preview]   |
-|   **Linux (x64)** (for glibc based OS)  |   [![][linux-x64-badge-master]][linux-x64-version-master]<br>[tar.gz][linux-x64-targz-master]<br>[Symbols (tar.gz)][linux-x64-symbols-targz-master]   |   N/A   |   N/A   |
+|   **Windows (x64)**    |   [![][win-x64-badge-master]][win-x64-version-master]<br>[Installer][win-x64-installer-master]<br>[zip][win-x64-zip-master]<br>[Symbols (zip)][win-x64-symbols-zip-master]<hr><b>Portable Build</b><br>[Installer][win-x64-installer-master]<br>[zip][win-x64-zip-master]<br>[Symbols (zip)][win-x64-symbols-zip-master]   |   [![][win-x64-badge-1.1.X]][win-x64-version-1.1.X]<br>[Installer][win-x64-installer-1.1.X]<br>[zip][win-x64-zip-1.1.X]   |   [![][win-x64-badge-preview]][win-x64-version-preview]<br>[Installer][win-x64-installer-preview]<br>[zip][win-x64-zip-preview]   |
```

Comment:
We should do that once we close on pulling the plug on distro specific binaries.